ament_cmake_core/cmake/core/all.cmake

```diff
@@ -23,38 +23,6 @@ if(NOT DEFINED ament_cmake_core_DIR)
   message(FATAL_ERROR "ament_cmake_core_DIR is not set")
 endif()
 
-# enable all new policies (if available)
-macro(_set_cmake_policy_to_new_if_available policy)
-  if(POLICY ${policy})
-    cmake_policy(SET ${policy} NEW)
-  endif()
-endmacro()
-_set_cmake_policy_to_new_if_available(CMP0000)
-_set_cmake_policy_to_new_if_available(CMP0001)
-_set_cmake_policy_to_new_if_available(CMP0002)
-_set_cmake_policy_to_new_if_available(CMP0003)
-_set_cmake_policy_to_new_if_available(CMP0004)
-_set_cmake_policy_to_new_if_available(CMP0005)
-_set_cmake_policy_to_new_if_available(CMP0006)
-_set_cmake_policy_to_new_if_available(CMP0007)
-_set_cmake_policy_to_new_if_available(CMP0008)
-_set_cmake_policy_to_new_if_available(CMP0009)
-_set_cmake_policy_to_new_if_available(CMP0010)
-_set_cmake_policy_to_new_if_available(CMP0011)
-_set_cmake_policy_to_new_if_available(CMP0012)
-_set_cmake_policy_to_new_if_available(CMP0013)
-_set_cmake_policy_to_new_if_available(CMP0014)
-_set_cmake_policy_to_new_if_available(CMP0015)
-_set_cmake_policy_to_new_if_available(CMP0016)
-_set_cmake_policy_to_new_if_available(CMP0017)
-_set_cmake_policy_to_new_if_available(CMP0018)
-_set_cmake_policy_to_new_if_available(CMP0019)
-_set_cmake_policy_to_new_if_available(CMP0020)
-_set_cmake_policy_to_new_if_available(CMP0021)
-_set_cmake_policy_to_new_if_available(CMP0022)
-_set_cmake_policy_to_new_if_available(CMP0023)
-_set_cmake_policy_to_new_if_available(CMP0042)
-
```

Comment:
Yes, all policies already available in the CMake version specified as minimum are enabled by default.